lib/src/components/NxSmallTag/NxSmallTag.scss

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@use '../../scss-shared/nx-text-helpers';
+
+.nx-small-tag {
+  @include nx-text-helpers.semi-bold;
+  $height: 24px;
+
+  align-items: center;
+  display: inline-flex;
+  height: calc(1em * var(--nx-line-height));
+  margin: 0 var(--nx-spacing-2x);
+  vertical-align: top;
+
+  + .nx-small-tag {
+    margin-left: 0;
+  }
```

Comment:
I did notice this affected the margin when multiple small tags are included in the paragraph(not in sequence, some text in between), there is no margin on the  second small tag, seems like the `+` css selector ignored the plain text. 
In this case, should we move these to the gallery example css file and apply the styles within the example block instead?
<img width="216" alt="Screenshot 2023-06-05 at 6 43 12 PM" src="https://github.com/sonatype/sonatype-react-shared-components/assets/37988395/1db94e9b-70a5-46b4-b2c7-0893e810d589">

Reply:
Ah right, updated here 6907f882554eaab2d08605229ad73180402d1b9f

Reply:
> Oh but you also need to remove the text whitespace before and after each NxSmallTag in that paragraph, since they handle their spacing with margin. If you want you could even mention that in the example description.

I was wrong about this.